cmd/collector/app/span_processor_test.go

```diff
@@ -37,8 +37,12 @@ import (
 	zc "github.com/jaegertracing/jaeger/thrift-gen/zipkincore"
 )
 
-var _ (io.Closer) = (*fakeSpanWriter)(nil)
-var blackListedService = "zoidberg"
+var (
+	_ io.Closer = (*fakeSpanWriter)(nil)
+	_ io.Closer = (processor.SpanProcessor)(nil)
```

Comment:
`processor.SpanProcessor` is an interface right? That will always pass this test. Should we check `app.spanProcessor`?